app/assets/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.js

```diff
@@ -0,0 +1,43 @@
+/* global GOVUK, $, ga */
+
+(function () {
+  'use strict'
+  window.GOVUK = window.GOVUK || {}
+
+  var ExplicitCrossDomainLinks = function () {
+    this.decorateForm = function (element, trackers) {
+      if (trackers.length) {
+        var existing = element.attr('action')
+        var linker = new window.gaplugins.Linker(trackers[0])
+        var tracked = linker.decorate(existing)
+        element.attr('action', tracked)
+      }
+    }
+
+    this.decorateLink = function (element, trackers) {
+      if (trackers.length) {
+        var existing = element.attr('href')
+        var linker = new window.gaplugins.Linker(trackers[0])
+        var tracked = linker.decorate(existing)
+        element.attr('href', tracked)
+      }
+    }
+  }
+
+  ExplicitCrossDomainLinks.start = function () {
```

Comment:
This doesn't quite match our normal structure for a GOVUK modules bit of code, but I'm aware I might be missing out on some previous discussions around this. Was it that there were some problems making this into a proper GOVUK module so this was the best solution for now?